components/ColonyQuestions.js

```diff
@@ -15,7 +16,18 @@ const StyledQuestionLabel = styled.label`
   text-align: center;
 `;
 
-export default function ColonyQuestions() {
+export default function ColonyQuestions({ currentColony }) {
+  const [formData, setFormData] = useLocalStorageState(
+    `formData_${currentColony?.id}`,
+    { defaultValue: [] }
+  );
```

Comment:
Just one thing: The defaultValue here is an array but it will never become an array. Each time, you submit a new form (from a different colony) you create a new localstorage entry and when you have 100 forms you will have 100 entries (with different names) in the local-storage. That is fine for the moment and you don't have to fix this. But the better way is to have an array with all entries including the colonyId to with the data belongs. If you later will migrate to a database you will have to think about this point. For now it is okay.

Reply:
Alright, thanks for the hint, I will keep that in mind.